### DISCUSSION TITLE
как применить агрументы

### DISCUSSION THREAD

**Discussion by Sofutikku** [bol-van/zapret#996] [Q&A]
когда использовал windows использовал zapret от Flowseal с этими агрументами
`
start "zapret: general" /min "%BIN%winws.exe" --wf-tcp=80,443 --wf-udp=443,50000-50100 ^
--filter-udp=443 --hostlist="list-general.txt" --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-fake-quic="%BIN%quic_initial_www_google_com.bin" --new ^
--filter-udp=50000-50100 --ipset="ipset-discord.txt" --dpi-desync=fake --dpi-desync-any-protocol --dpi-desync-cutoff=d3 --dpi-desync-repeats=6 --new ^
--filter-tcp=80 --hostlist="list-general.txt" --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^
--filter-tcp=443 --hostlist="list-general.txt" --dpi-desync=split --dpi-desync-split-pos=1 --dpi-desync-autottl --dpi-desync-fooling=badseq --dpi-desync-repeats=8`
как можно запустить эти аргументы на linux
(извиняюсь за ламерский вопрос)

**Comment by 8hrsk** [comment]
В конфиг пихаешь и всё...
Это аргументы для nfqws. Нужно будет закинуть хостлист (если он нужен) в запрет, а так же заменить путь к бинарникам

**Reply by bol-van** [reply]
Закидывать в /opt/zapret любые свои файлы не рекомендовано, потому что они сносятся инсталятором при обновлении

**Reply by Sofutikku** [reply]
> В конфиг пихаешь и всё... Это аргументы для nfqws. Нужно будет закинуть хостлист (если он нужен) в запрет, а так же заменить путь к бинарникам

а что означает --wf-tcp=80,443?, я не нашел про него инфу в документации

**Reply by bol-van** [reply]
windivert filter
аналог iptables
что перехватывать, с чем будет работать winws

**Reply by Sofutikku** [reply]
> windivert filter аналог iptables что перехватывать, с чем будет работать winws

а как это в конфиге записать? 
и вообще стоит ли

**Reply by bol-van** [reply]
На linux этих параметров нет, перехват выполняется через ip/nf tables, настраивается в /opt/zapret/config отдельными переменными
